README.md

```diff
@@ -1,4 +1,55 @@
+![puta](https://github.com/PactCoffee/js-ui-challenge/raw/master/puta.png)
+
 Pact's front end challenge
 ==========================
 
-TBA
+The general idea
+----------------
+
+We have a sign up flow affectionately known as "The Funnel". It's primary use is to sign up customers to our service. They enter their information, it gets sent to the server. **The challenge is to build a very simple version of [our current funnel](http://pactcoffee.com/funnel/intro)**. Sounds easy enough, right?
+
+
+But wait! There's more...
+-------------------------
+
+1. The funnel is separated into "stages". Each stage collects certain information from the user. For this challenge, those stages are:
+  - Name & email address
+  - Brew type selection
+  - Delivery address
+  - Confirmation of everything
```

Comment:
What do you think it's for?

(Deliberate question — I want the readme to have just-enough-detail)